src/lib/classes/lsp7-digtal-asset.spec.ts

```diff
@@ -232,4 +233,50 @@ describe('LSP7DigitalAsset', () => {
       expect(ethers.utils.toUtf8String(retrievedSymbol)).toEqual(symbol);
     });
   });
+  describe('deploy lsp7 with specified creators', () => {
+    let digitalAsset: LSP7Mintable;
+    const controllerAddress = '0xaDa25A4424b08F5337DacD619D4bCb21536a9B95';
+    const name = 'TOKEN';
+    const symbol = 'TKN';
+    const isNFT = true;
+    const creators = [
+      '0xFCA72D5763b8cFc686C2285099D5F35a2F094E9f',
+      '0x591c236982b089Ad4B60758C075fA50Ec53CD674',
+    ];
+
+    it('should deploy with specified creators', async () => {
+      const lspFactory = new LSPFactory(provider, signer);
+      const lsp7DigitalAsset = (await lspFactory.LSP7DigitalAsset.deploy({
+        controllerAddress,
+        name,
+        symbol,
+        creators,
+        isNFT,
+      })) as DeployedLSP7DigitalAsset;
+
+      expect(lsp7DigitalAsset.LSP7DigitalAsset.address).toBeDefined();
+      expect(Object.keys(lsp7DigitalAsset).length).toEqual(2);
+
+      digitalAsset = LSP7Mintable__factory.connect(
+        lsp7DigitalAsset.LSP7DigitalAsset.address,
+        signer
+      );
+    });
+    it('should have LSP4Creators[] set correctly', async () => {
+      const [creatorArrayLength] = await digitalAsset.getData([LSP4_KEYS.LSP4_CREATORS_ARRAY]);
+      expect(creatorArrayLength).toEqual(
+        '0x0000000000000000000000000000000000000000000000000000000000000002'
+      );
+
+      const [creator1, creator2] = await digitalAsset.getData([
+        LSP4_KEYS.LSP4_CREATORS_ARRAY.slice(0, 34) +
```

Comment:
@Hugoo I think it would be very handy to have this helper function in `erc725.js` ! :)